pkg/Library/Mediapipe/HolisticSticker.js

```diff
@@ -5,21 +5,24 @@
  * license that can be found in the LICENSE file.
  */
 ({
-async initialize(_, state, {service}) {
+async initialize({index}, state, {service}) {
```

Comment:
nit - `index` is not used in `initialize`

Reply:
I think we will want to use it soon.